tests/ga/test_extension.py

```diff
@@ -3411,6 +3398,111 @@ def manifest_location_handler(url, **kwargs):
                 protocol.client.fetch_manifest(ext_handlers.extHandlers[0].versionUris,
                     timeout_in_minutes=0, timeout_in_ms=200)
 
+class TestExtensionWithMockHTTP(AgentTestCase):
+
+    def setUp(self):
+        AgentTestCase.setUp(self)
+
+    def tearDown(self):
+        AgentTestCase.tearDown(self)
+
+    @contextlib.contextmanager
+    def _setup_test_env(self, test_data):
```

Comment:
in this case the context manager is not buying you anything, I think. You could do the setup in the test itself (or are you planning to re-use the context manager for additional tests?)

Reply:
Addressed it